app/grandchallenge/pages/views.py

```diff
@@ -159,6 +159,7 @@ def get_average_job_duration_for_phase(phase):
         algorithm_image__pk__in=algorithm_images,
         status=Job.SUCCESS,
         creator=None,
```

Comment:
Ah yes you're right. Actually, we are now also including jobs that ran if the algorithm is linked to the archive, so not only challenge submissions. Further subsetting by viewer_groups does not mitigate that though, because challenge admins and archive editors are identical in that case, but I guess that's ok.

Reply:
Hmm, you're right. That might be less common that submitting to multiple phases (which almost always happens). The CIV is related to the evaluation too, so maybe you could do something like `inputs__evaluations__submission__phase=phase,`?

Reply:
We might also need a `.distinct()` on the jobs query.

Reply:
Ah yes that's a good idea. Querying for `outputs__evaluations__submission__phase=phase` doesn't work, but we can retrieve the evaluation input civ pks for the phase and then query `outputs__pk`.

Reply:
Ah I misinterpreted the error messages I got. It's possible, but it needs to be `outputs__evaluation_evaluations_as_input__submission__phase`